lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -424,7 +428,8 @@ def new_fixed_axis(self, loc,
                        ):
 
         if axes is None:
-            warnings.warn("'new_fixed_axis' explicitly requires the axes keyword.")
+            warnings.warn(
```

Comment:
should this be `cbook._warn_external`?

Reply:
I have a separate PR (in the works) doing some massive change of warnings.warn to warn_external.